src/components/popup/details/vendors/vendor.jsx

```diff
@@ -0,0 +1,52 @@
+import {h, Component} from "preact";
+
+import style from "./vendors.less";
+import ExternalLinkIcon from "../../../externallinkicon/externallinkicon";
+import Label from "../../../label/label";
+
+class LocalLabel extends Label {
+	static defaultProps = {
+		prefix: 'vendors'
+	};
+}
+
+export default class Vendor extends Component {
+	shouldComponentUpdate() {
```

Comment:
Maybe it is worth to mention that there is a possibility to check if new props are different than current - https://reactjs.org/docs/optimizing-performance.html#shouldcomponentupdate-in-action. Did you consider using it?

Reply:
Yes, but I wonder in which case these attributes can be updated after first render